### PR TITLE
add phpinsights

### LINKED ISSUE #16: Configure PHP Insights
https://phpinsights.com/get-started.html

### PR DESCRIPTION
**Pull request by rtio**
it closes #16

### CODE REVIEW

File: phpinsights.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    'exclude' => [
+        'src/Migrations',
+        'src/Kernel.php',
+        'phpinsights.php',
+    ],
+
+    'add' => [
+        //  ExampleMetric::class => [
+        //      ExampleInsight::class,
+        //  ]
```
Comment (ianrodrigues) [inline phpinsights.php]: not needed

File: phpinsights.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    ],
+
+    'remove' => [
+        //  ExampleInsight::class,
```
Comment (ianrodrigues) [inline phpinsights.php]: not needed

File: phpinsights.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    ],
+
+    'config' => [
+        //  ExampleInsight::class => [
+        //      'key' => 'value',
+        //  ],
```
Comment (ianrodrigues) [inline phpinsights.php]: not needed

File: phpinsights.php (hunk 1 above)
Comment (rtio) [reply phpinsights.php]: removed

File: phpinsights.php (hunk 2 above)
Comment (rtio) [reply phpinsights.php]: removed

File: phpinsights.php (hunk 3 above)
Comment (rtio) [reply phpinsights.php]: removed

**Pull request merged**